autogen/safer-cluster/main.tf.tmpl

```diff
@@ -34,14 +34,16 @@ module "gke" {
 
   // We need to enforce a minimum Kubernetes Version to ensure
   // that the necessary security features are enabled.
-  kubernetes_version = "latest"
+  kubernetes_version = var.kubernetes_version
 
   // Nodes are created with a default version. The nodepool enables
   // auto_upgrade so that the node versions can be kept up to date with
   // the master upgrades.
   //
   // https://cloud.google.com/kubernetes-engine/versioning-and-upgrades
-  node_version = ""
+  node_version = var.node_version
```

Comment:
@morgante Is node_version actually used anywhere in any of the modules? I what grepping through the code last night, but AFAICT, node_version is never consumed by any resource. Am I crazy, or can we just remove node_version, node_version_regional, and node_version_zonal from autogen/main & autogen/safer-cluster?

Reply:
You're right, it looks like we might have dropped it entirely. Feel free to remove the variable in that case.